third_party/move/move-compiler-v2/src/pipeline/ability_checker.rs

```diff
@@ -97,6 +97,25 @@ fn cond_check_drop_for_temp_with_msg(
     cond_check_drop(func_target, code_offset, ty, loc, err_msg)
 }
 
+/// Conditionally checks for drop abilit as `cond_check_drop_for_temp_with_msg`
+/// `op_name`: name of the operation which requires drop ability for the given temp `t`
+fn cond_check_drop_for_temp_by(
+    target: &FunctionTarget,
+    code_offset: CodeOffset,
+    t: TempIndex,
+    loc: &Loc,
+    op_name: &str,
+) {
+    let name_of_t = target.get_local_name_for_error_message(t);
+    cond_check_drop_for_temp_with_msg(
+        target,
+        code_offset,
+        t,
+        loc,
+        &format!("`{op_name}` requires drop ability for {name_of_t}"),
```

Comment:
You need back-quotes around the second formatting placeholder in the string.

Personally, I **hate** the named placeholder feature of `format!`, since it means that I have to pay attention to the contents of things in `".."` to understand value flows.  I really really prefer
```
        &format!("`{}` requires drop ability for `{}`", op_name, name_of_t);
```
since I can ignore the contents of `".."` unless I care about what's being logged/printed.